src/main/java/com/softserveinc/ita/utils/ElementsUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.softserveinc.ita.utils;
+
+import com.codeborne.selenide.SelenideElement;
+import lombok.experimental.UtilityClass;
+import org.openqa.selenium.support.ui.Select;
+
+@UtilityClass
+public class ElementsUtil {
+
+    public SelenideElement setText(SelenideElement elementSelector, String text) {
+        var input = elementSelector;
+        input.click();
+        input.sendKeys(text);
+
+        return input;
+    }
+
+    public void selectModalMenu(SelenideElement selector , String text) {
+        var selectForm = selector;
```

Comment:
same, you have this element in arguments

Reply:
its deprecated class